AjaxControlToolkit.Jasmine/Suites/SlideShowTests/SlideShowTests.aspx

```diff
@@ -91,7 +91,6 @@
 
             it("enables/disables prev and next buttons after play/stop button pressed", function(done) {
                 var that = this;
-                this.$playStopButton.click();
 
                 setTimeout(function() {
```

Comment:
how about elimitaing this fixed timeout too? what condition is being waited for here?